modules/ROOT/pages/api-sync.adoc

```diff
@@ -1,71 +1,50 @@
-= API Sync
+= Sync Your API Projects with API Sync
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, anypoint platform, sync, api sync
+:keywords: api, anypoint platform, sync, api sync, MuleSoft VCS
 
-The API Sync feature of Anypoint Studio enables you to:
+With API Sync, you can:
 
-* Start designing a RAML, or OAS 2.0 and 3.0 API specification in Studio and sync it with MuleSoft VCS.
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
+* Edit the API spec you imported from Exchange, and push your changes to MuleSoft VCS.
+* Use API Sync on more than one API spec to scaffold multiple flows in different Mule configuration files.
+* Monitor the API versions that you implement in your project and update your flows based on the latest published version.
+* Pull a RAML or OAS 2.0 and 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and pull the updates back to MuleSoft VCS.
+* Develop your Mule applications following API Lifecycle development practices.
 
-API Sync supports importing both RAML, OAS 2.0 and OAS 3.0 specifications.
+API Sync supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
```

Comment:
```suggestion
API Sync supports importing RAML, OAS 2.0, and OAS 3.0 specs, is bundled by default with Studio, and requires Mule 4.1.4 and later.
```

Reply:
Do you need the first bit? Isn't it obvious from the previous bullets? 

API Sync is bundled by default with Studio and requires Mule 4.1.4 or later.

Reply:
Importing and pulling may be obvious to someone that understands the function of a source control system.
As this is the introduction of API Sync, it may be somehow useful to mention that API Sync can import and pull specs. Thanks!

Reply:
OK, then suggest rewording to separate the supports from the other bits. As it is, it's kind of a mishmash of thoughts.

bundled and requires
supports

Reply:
I'll create a WI to work on that! 
Thanks!